src/MagnumPlugins/DrFlacAudioImporter/DrFlacImporter.cpp

```diff
@@ -35,6 +35,42 @@
 
 namespace Magnum { namespace Audio {
 
+namespace {
+
+    typedef Buffer B;
+
+    // number of channels = 8
+    // number of bits = 5
+    const B::Format  flacFormatTable[9][5] = {
+        { B::Format::Mono8,   B::Format::Mono8,   B::Format::Mono16,   B::Format::MonoFloat,   B::Format::MonoDouble   },   // None
+        { B::Format::Mono8,   B::Format::Mono8,   B::Format::Mono16,   B::Format::MonoFloat,   B::Format::MonoDouble   },   // Mono
+        { B::Format::Stereo8, B::Format::Stereo8, B::Format::Stereo16, B::Format::StereoFloat, B::Format::StereoDouble },   // Stereo
+        { B::Format::Quad8, B::Format::Quad8, B::Format::Quad16, B::Format::Quad32, B::Format::Quad32 },                    // Not a thing
+        { B::Format::Quad8, B::Format::Quad8, B::Format::Quad16, B::Format::Quad32, B::Format::Quad32 },                    // Quad
+        { B::Format::Quad8, B::Format::Quad8, B::Format::Quad16, B::Format::Quad32, B::Format::Quad32 },                    // Also not a thing
+        { B::Format::Surround51Channel8, B::Format::Surround51Channel8, B::Format::Surround51Channel16, B::Format::Surround51Channel32, B::Format::Surround51Channel32 }, // 5.1
+        { B::Format::Surround61Channel8, B::Format::Surround61Channel8, B::Format::Surround61Channel16, B::Format::Surround61Channel32, B::Format::Surround61Channel32 }, // 6.1
+        { B::Format::Surround71Channel8, B::Format::Surround71Channel8, B::Format::Surround71Channel16, B::Format::Surround71Channel32, B::Format::Surround71Channel32 }  // 7.1
```

Comment:
Ah, right. Now I understand :)

(Sorry if my comments sound angry, mean and/or grumpy today, I'm completely overloaded with work and it's killing me.)